python/ray/serve/_private/constants.py

```diff
@@ -163,6 +163,8 @@
     "instead (see https://docs.ray.io/en/latest/serve/model_composition.html)."
 )
 
+# Environment variable name for to specify the encoding of the log messages
+RAY_SERVE_LOG_ENCODING_ENV = "RAY_SERVE_LOG_ENCODING_ENV"
```

Comment:
Drop the `_ENV` suffix; we don't use that for any of the other environment variable flags